### ISSUE TITLE
Error: Cannot find module 'jscodeshift'

### ISSUE THREAD

**Issue by fbryo21** [facebook/jscodeshift#441]
```
PS C:\PROYEK\PIECLO> jscodeshift -t node_modules/@material-ui/codemod/lib/v4.0.0/top-level-imports.js ./components/appBar/Account.tsx
Processing 1 files...
Spawning 1 workers...
Sending 1 files to free worker...
internal/modules/cjs/loader.js:968
  throw err;
  ^

Error: Cannot find module 'jscodeshift'
Require stack:
- C:\PROYEK\PIECLO\node_modules\jscodeshift-add-imports\index.js
- C:\PROYEK\PIECLO\node_modules\@material-ui\codemod\lib\v4.0.0\top-level-imports.js
- C:\Users\User\AppData\Roaming\npm\node_modules\jscodeshift\src\Worker.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\PROYEK\PIECLO\node_modules\jscodeshift-add-imports\/index.js:1:11)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Module._compile (C:\Users\User\AppData\Roaming\npm\node_modules\jscodeshift\node_modules\pirates\lib\index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Object.newLoader [as .js] (C:\Users\User\AppData\Roaming\npm\node_modules\jscodeshift\node_modules\pirates\lib\index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:985:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\PROYEK\\PIECLO\\node_modules\\jscodeshift-add-imports\\index.js',
    'C:\\PROYEK\\PIECLO\\node_modules\\@material-ui\\codemod\\lib\\v4.0.0\\top-level-imports.js',
    'C:\\Users\\User\\AppData\\Roaming\\npm\\node_modules\\jscodeshift\\src\\Worker.js'
  ]
}
All done.
0 errors
0 unmodified
0 skipped
0 ok
Time elapsed: 0.983seconds
```
Even though I have installed jscodeshift, but the error still appears

STACKS: Next.js, Express, Material-UI

**Comment by oliviertassinari** [comment]
It's getting fixed in https://github.com/mui-org/material-ui/pull/26932

**Comment by AllieCR** [comment]
> It's getting fixed in [mui-org/material-ui#26932](https://github.com/mui-org/material-ui/pull/26932)

That PR link currently leads to a 500 error page

**Comment by oliviertassinari** [comment]
It's fixed now. This issue can be fixed
